app/koi-client/src/page/@backoffice@screen@[partyId]/component/StockScreen/index.tsx

```diff
@@ -25,9 +25,19 @@ export default function StockScreen({ party }: Props) {
 
   const startedTime = dayjs(stock?.startedTime).toDate();
   const isTransaction = stock?.isTransaction ?? false;
+  const [time, setTime] = useState('');
```

Comment:
time 상태의 초기값을 충분하게 나타낼 수 있음에도 ''로 초기화하는 것은 부적절합니다.
time 상태의 초기값을 나타내는 방법은 다음과 같습니다.

1. StockScreen 바깥에 함수를 만듭니다.
```
const getTimeDistanceWithCurrent = (date: Date) => {
  const { seconds, minutes } = getDateDistance(date, new Date());
  return `${prependZero(minutes, 2)}:${prependZero(seconds, 2)}`;
};
```

2. 다음과 같이 수정합니다.
```suggestion
  const [time, setTime] = useState(() => {
    return getTimeDistanceWithCurrent(startedTime);
  });
```

Reply:
오호 수정했습니다!